contracts/SplitEquippable/README.md

```diff
@@ -709,274 +709,253 @@ async function mintTokens(
+
+  // Accept 3 gems for each kanaria
+  console.log("Accepting Gems");
+  for (let tokenId = 1; tokenId <= totalBirds; tokenId++) {
+    allTx = [
+      await kanaria.acceptChild(tokenId, 2, gem.address, 3 * tokenId),
+      await kanaria.acceptChild(tokenId, 1, gem.address, 3 * tokenId - 1),
+      await kanaria.acceptChild(tokenId, 0, gem.address, 3 * tokenId - 2),
+    ];
   }
+  await Promise.all(allTx.map((tx) => tx.wait()));
+  console.log(`Accepted gems for each kanaria`);
 }
 ````
 
 In order for the `mintTokens` to be called, we have to add it to the `main` function:
 
 ````typescript
-  await mintTokens(kanariaNesting, gemNesting);
+  await mintTokens(kanariaNestable, gemNestable);
 ````
 
-Having minted both `Kanaria`s and `Gem`s, we can now add resources to them. The resources are added to the
-`SimpleExternalEquip` parts of them. We will add resources to the `Kanaria` using the `addKanariaResources` function.
-It accepts `Kanaria` and address of the `Base` smart contract. Resources will be added using the
-[`addResourceEntry`](#addresourceentry) method. We will add a default resource, which doesn't need a `baseAddress`
-value. The composed resource needs to have the `baseAddress`. We also specify the fixed parts IDs for background, head,
-body and wings. Additionally we allow the gems to be equipped in the slot parts IDs. With the resource entires added,
+Having minted both `Kanaria`s and `Gem`s, we can now add assets to them. The assets are added to the
+`SimpleExternalEquip` parts of them. We will add assets to the `Kanaria` using the `addKanariaAssets` function.
+It accepts `Kanaria` and address of the `Base` smart contract. Assets will be added using the
+[`addAssetEntry`](#addassetentry) method. We will add a default asset, which doesn't need a `baseAddress`
+value. The composed asset needs to have the `baseAddress`. We also specify the fixed parts IDs for background, head,
+body and wings. Additionally we allow the gems to be equipped in the slot parts IDs. With the asset entires added,
 we can add them to a token and then accept them as well:
 
 ````typescript
-async function addKanariaResources(
+async function addKanariaAssets(
   kanaria: SimpleExternalEquip,
   baseAddress: string
 ): Promise<void> {
-  const resourceDefaultId = 1;
-  const resourceComposedId = 2;
+  const assetDefaultId = 1;
+  const assetComposedId = 2;
   let allTx: ContractTransaction[] = [];
-  let tx = await kanaria.addResourceEntry(
-    {
-      id: resourceDefaultId,
-      equippableGroupId: 0, // Only used for resources meant to equip into others
-      baseAddress: ethers.constants.AddressZero, // base is not needed here
-      metadataURI: "ipfs://default.png",
-    },
+  let allTx: ContractTransaction[] = [];
+  let tx = await kanaria.addAssetEntry(
+    0, // Only used for assets meant to equip into others
+    ethers.constants.AddressZero, // base is not needed here
+    "ipfs://default.png",
```

Comment:
I'd add the keys to which the values are passed for more clarity to a first-time reader:

```suggestion
    equippableGroupId: 0, // Only used for assets meant to equip into others
    baseAddress: ethers.constants.AddressZero, // base is not needed here
    metadataURI: "ipfs://default.png",
```